src/api/java/com/minecolonies/api/entity/citizen/AbstractEntityCitizen.java

```diff
@@ -115,8 +115,7 @@ public int getTicksExisted()
      * @param dimensionIn dimension to travel to.
      */
     @Override
-    @Nullable
-    public Entity changeDimension(final int dimensionIn)
+    public Entity changeDimension(int dimensionIn, net.minecraftforge.common.util.ITeleporter teleporter)
```

Comment:
Can't we import the ITeleporter for this?

Reply:
ye we can, just had that copied